CONTRIBUTING.md

```diff
@@ -47,7 +46,7 @@ cd mainflux
 git remote add upstream https://github.com/mainflux/mainflux.git
 ```
 
-If you cloned a while ago, get the latest changes from upstream:
+If you cloned repository is behind the upstream commits than get the latest changes from upstream:
```

Comment:
`If your cloned repository is behind the upstream commits, then get the latest changes from upstream:`

Reply:
Took care of this. Thanks @anovakovic01